File: app/src/main/java/com/example/piggybank/CajaFuerteFragment.java
package com.example.piggybank;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

public class CajaFuerteFragment extends Fragment {
    private Button Sumar;
    private Button Restar;
    private Button Guardar;
    private Button Atras;
    public CajaFuerteFragment() {

    }


    public static CajaFuerteFragment newInstance() {
        CajaFuerteFragment fragment = new CajaFuerteFragment();
        Bundle args = new Bundle();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_caja_fuerte, container, false);
    }

    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        Sumar = view.findViewById(R.id.Sumar);
        Sumar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                System.out.println("Se sumo con exito");
            }
        });
        Restar = view.findViewById(R.id.Restar);
        Restar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                System.out.println("Se resto con exito");
            }
        });
        Guardar = view.findViewById(R.id.Guardar);
        Guardar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                System.out.println("Se guardo");
            }
        });
        Atras = view.findViewById(R.id.Atras);
        Atras.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                System.out.println("Chao pescao");
                FragmentManager fragmentManager = getFragmentManager();
                FragmentTransaction fragmentTransaction= fragmentManager.beginTransaction();
                fragmentTransaction.replace(R.id.container, MenuFragment.newInstance());
                fragmentTransaction.commit();
            }
        });

    }
}

File: app/src/main/java/com/example/piggybank/CajaFuerteFragment.kt
package com.example.piggybank

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import kotlinx.android.synthetic.main.fragment_caja_fuerte.Atras;
import kotlinx.android.synthetic.main.fragment_caja_fuerte.Sumar;
import kotlinx.android.synthetic.main.fragment_caja_fuerte.Restar;
import kotlinx.android.synthetic.main.fragment_caja_fuerte.Guardar;

class CajaFuerteFragment : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_caja_fuerte, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        Sumar?.setOnClickListener(View.OnClickListener { println("Se sumo con exito") })

        Restar?.setOnClickListener(View.OnClickListener { println("Se resto con exito") })

        Guardar?.setOnClickListener(View.OnClickListener { println("Se guardo") })

        Atras?.setOnClickListener(View.OnClickListener {
            println("Chao pescao")
            val fragmentManager = fragmentManager
            val fragmentTransaction = fragmentManager!!.beginTransaction()
            fragmentTransaction.replace(R.id.container, MenuFragment.newInstance())
            fragmentTransaction.addToBackStack("Last Transaction")
            fragmentTransaction.commit()
        })
    }

    companion object {
        @JvmStatic
        fun newInstance(): CajaFuerteFragment {
            val fragment = CajaFuerteFragment()
            val args = Bundle()
            fragment.arguments = args
            return fragment
        }
    }
}